config/m4/cma.m4

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (C) UT-Battelle, LLC. 2014-2015. ALL RIGHTS RESERVED.
+# $COPYRIGHT$
+# $HEADER$
+#
+cma_happy="no"
+AC_ARG_ENABLE([cma],
```

Comment:
it should check for cma headers presence:

AC_CHECK_HEADERS([sys/uio.h])

or in configure.ac